tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -271,15 +271,18 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                                    'value. When a list of inputs is overridden by the --input ' +
                                    'parameter, this scale ' +
                                    'is not applied for any input that does not match with ' +
-                                   'the original input of the model.')
+                                   'the original input of the model.' +
+                                   'If both --mean and --scale  are specified, ' +
+                                   'the mean is subtracted first and then scale is applied ' +
+                                   'regardless the order of options in command line.')
```

Comment:
please update this help in `Converting_Model.md` for these options. We have a copy of this text there.

Reply:
done